resolwe_bio/processes/variant_calling/create_genomicsdb.py

```diff
@@ -0,0 +1,198 @@
+"""Run GATK GenomicsDBImport tool."""
+import shutil
+from pathlib import Path
+
+from plumbum import TEE
+
+from resolwe.process import (
+    BooleanField,
+    Cmd,
+    DataField,
+    FileField,
+    GroupField,
+    IntegerField,
+    ListField,
+    Process,
+    SchedulingClass,
+    StringField,
+)
+from resolwe.process.fields import DirField
+
+
+class GatkGenotypeGVCFs(Process):
+    """Import single-sample GVCFs into GenomicsDB before joint genotyping."""
+
+    slug = "gatk-genomicsdb-import"
+    name = "GATK GenomicsDBImport"
+    category = "GATK"
+    process_type = "data:genomicsdb"
+    version = "1.0.0"
+    scheduling_class = SchedulingClass.BATCH
+    requirements = {
+        "expression-engine": "jinja",
+        "executor": {
+            "docker": {"image": "public.ecr.aws/s4q6j6e8/resolwebio/dnaseq:6.0.0"}
+        },
+        "resources": {
+            "cores": 4,
+            "memory": 32768,
+            "storage": 64,
```

Comment:
Maybe it's best to start with a larger storage number (200). `64` will allow testing with a few samples.